index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Identities
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class="remove">
+    "use strict";
+    // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+    var respecConfig = {
+      github: "WICG/digital-credentials",
+      editors: [
+        {
+          name: "Marcos Caceres",
+          email: "marcosc@apple.com",
+          company: "Apple Inc.",
+          companyURL: "https://apple.com",
+        },
+        {
+          name: "Sam Goto",
+          email: "goto@google.com",
+          company: "Google Inc.",
+          companyURL: "https://google.com",
+        },
+        // Add additional editors here.
+        // https://github.com/w3c/respec/wiki/editors
+      ],
+      shortName: "digital-identity",
+      specStatus: "CG-DRAFT",
+      group: "wicg",
+      localBiblio: {
+      },
+      xref: {
+        profile: "web-platform",
+      },
+    };
+    </script>
+  </head>
+  <body data-cite="credential-management-1 WebCryptoAPI">
+    <section id="abstract">
+      <p>
+        This document specifies an API to allow user agents to mediate access
+        to, and representing, a digital identity (e.g. a digital driver's
+        license). The API builds on [[[credential-management-1]]] as a means to
+        request a digital identity from the user agent or underlying platform.
+      </p>
+    </section>
```

Comment:
Maybe a big example up front would help explain?

```suggestion
<div class=example>
Example showing how a website can request a digital identity.

```javascript
let {protocol, data} = await navigator.identity.get({
  signal: controller,
  mediation: "optional", 
  providers: [{
    protocol: "fake-protocol",
    request: "{type: 'drivers-license'}",
    nonce: "0xCAFE",
  }]
});
```
</div>

    </section>
```

Reply:
Definitely want to add examples, but I think we should hold till we have the first item in the registry. Then we can add real functional examples.

Reply:
Filed https://github.com/WICG/digital-credentials/issues/69

Reply:
Sure, that works too.